extensions/method-names/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "method-names"
```

Comment:
It should be `near-cli-method-names` to produce a binary that `near-cli-rs` can pick up when one calls `near-cli method-names`.